P5/Source/Specs/post.xml

```diff
@@ -0,0 +1,252 @@
+                <!--[Adapted from the <ref target="#TEIcmc2012"
+                    >DeRiK-Schema</ref>].-->
+            </desc>
+            <datatype minOccurs="1" maxOccurs="1">
+                <dataRef key="teidata.count"/>
+            </datatype>
+        </attDef>
+    </attList>
+    <exemplum type="Chat2CLARIN" xml:lang="de" versionDate="2021-09-07">
+        <p>standard user postings in chat</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <post type="standard" generatedBy="human" who="#A05" rend="color:red">
+                <p>oh ja, in die Wärme...</p>
+            </post>
+            <post type="standard" generatedBy="human" who="#A03" rend="color:red">
+                <p>nein, hast du nicht !!</p>
+            </post>
+        </egXML>
+    </exemplum>
+    <exemplum type="Chat2CLARIN" xml:lang="de" versionDate="2021-09-07">
+        <p>chat message-type "system": user enters a chatroom, "system" defined in listPerson</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <listPerson>
+                <person xml:id="system">
+                    <persName>system</persName>
+                </person>
+                <!-- [...] -->
+            </listPerson>
+            <!--  [...] -->
+            <post type="event" generatedBy="system" who="#system" rend="color:navy">
+                <p>
+                    <name type="nickname" corresp="#A07">Interseb</name> betritt den Raum.</p>
+            </post>
+        </egXML>
+    </exemplum>
+    <exemplum type="Chat2CLARIN" xml:lang="de" versionDate="2021-09-07">
+        <p>action message in chat</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <post type="event" generatedBy="human" who="#A02" rend="color:blue">
+                <p>
+                    <name type="nickname" corresp="#A02">McMike</name> hechtet zum Steuer </p>
+            </post>
+        </egXML>
+    </exemplum>
+    <exemplum type="Chat2CLARIN" xml:lang="de" versionDate="2021-09-07">
+        <p>chat messages incl. interaction words (types: inflective and acronym) and an interaction
+            phrase</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <post type="standard" generatedBy="human" who="#A03" rend="color:green">
+                <p>
+                    <w type="AKWIND">*</w><w type="AKW" subtype="inflective">schwank</w><w
+                        type="AKWIND">*</w>
+                    <!--...-->
+                    <w type="AKWIND">*</w><w type="AKW">seekrank</w><w type="AKWIND">*</w>
+                </p>
+            </post>
+            <!--[...] -->
+            <post type="standard" generatedBy="human" who="#A02" rend="color:blue">
+                <p>
+                    <w type="AKWIND">*</w><w type="AKW" subtype="acronym">lol</w><w type="AKWIND"
+                        >*</w> [...] </p>
+            </post>
+            <!--[...]-->
```

Comment:
please do not use the brackets here, just: `<!-- ... -->`

Reply:
fixed at f0e7d1e26.